src/compiler/checker.ts

```diff
@@ -3531,9 +3532,11 @@ namespace ts {
                         context.enclosingDeclaration = undefined;
                         if (getCheckFlags(propertySymbol) & CheckFlags.Late) {
                             const decl = first(propertySymbol.declarations);
-                            const name = hasLateBindableName(decl) && resolveEntityName(decl.name.expression, SymbolFlags.Value);
-                            if (name && context.tracker.trackSymbol) {
-                                context.tracker.trackSymbol(name, saveEnclosingDeclaration, SymbolFlags.Value);
+                            if (hasLateBindableName(decl)) {

```

Comment:
This is a copy of the primary change from #24668 - it's required to make the symbols get looked up correctly.